modules/openapi-generator/src/main/resources/dart/libraries/dio/serialization/json_serializable/class.mustache

```diff
@@ -46,7 +46,8 @@ class {{{classname}}} {
     {{#defaultValue}}defaultValue: {{{defaultValue}}},{{/defaultValue}}
     name: r'{{{baseName}}}',
     required: {{#required}}true{{/required}}{{^required}}false{{/required}},
-    includeIfNull: {{#required}}{{#isNullable}}true{{/isNullable}}{{^isNullable}}false{{/isNullable}}{{/required}}{{^required}}false{{/required}}
+    includeIfNull: {{#required}}{{#isNullable}}true{{/isNullable}}{{^isNullable}}false{{/isNullable}}{{/required}}{{^required}}false{{/required}},
+    {{#isEnumOrRef}}{{#enumUnknownDefaultCase}}unknownEnumValue: {{{datatypeWithEnum}}}.unknownDefaultOpenApi,{{/enumUnknownDefaultCase}}{{/isEnumOrRef}}
```

Comment:
can you use following instead?

```
    {{#isEnumOrRef}}
    {{#enumUnknownDefaultCase}}
    unknownEnumValue: {{{datatypeWithEnum}}}.unknownDefaultOpenApi,
    {{/enumUnknownDefaultCase}}
    {{/isEnumOrRef}}
```

this will allow the blank line